roles/azure_computing/README.md

```diff
@@ -0,0 +1,121 @@
+# Azure compute
+
+## Intro
+
+This is one of three Azure roles
+
+ - [azure_general](../azure_general/README.md) role creates Azure group which contains the network, VMs, network interfaces, security groups. This helps with an overview of used and available Azure resources
+ - [azure_networking](../azure_networking/README.md) role defines and mnages the entire stack of network: networks, sub-networks and security groups within the individual Azure group
+ - [azure_computing](../azure_computing/README.md) role defines and manages the VMs, their storage, network interfaces, public IP and private IPs inside the invididual Azure group
+
+## Defining variables
+
+### Static inventory
+
+The most important variables are in `static_inventory/[stack_name].yml`
+ - `azure_flavor` - this one triggers the generation of VMs
+ - `host_networks` - besides the `azure_flavor` this variables connects the VM to correct azure network
+
+for example
+ 
+```
+      hosts:
+        earl6:
+          azure_flavor: Standard_DS1_v2
+          host_networks:
+            - name: "{{ stack_prefix }}_internal_management"
+              security_group: "{{ stack_prefix }}_logservers"
+              assign_floating_ip: true
+```
+without these two variables the `azure.yml` playbook will fail.
+
+### Group variables
+
+Following variables must be also defined in the `group_vars/[stack_name]/vars.yml`
+
+ - `azure_networks` - must match the `host_networks` name from the `static_inventory/[stack_name].yml`
+ - `azure_image` - most of the time we use the same image for longer time, so if you are not sure, it's best to copy paste from another group
+ - (not urgent) `azure_resource_group` is by default already assigned to "{{ stack_name }}", therefore there is no need to define it here, but it could be changed
```

Comment:
```suggestion
 - (optional) `azure_resource_group` is by default already assigned to "{{ stack_name }}", therefore there is no need to define it here, but it could be changed
```